extensions/amp-bind/amp-bind.md

```diff
@@ -293,3 +321,42 @@ object:
 key_value:
   expr ':' expr
 ```
+
+## Debugging
+
+Testing in development mode (with the URL fragment `#development=1`) is recommended to clearly surface warnings and errors during development.
```

Comment:
Optional:

Test in development mode (with URL fragment `#development=1`) to clearly surface warnings and errors during development.